sources/packages/backend/apps/api/src/services/user/user.service.ts

```diff
@@ -1,8 +1,8 @@
 import { Injectable } from "@nestjs/common";
-import { SERVICE_ACCOUNT_DEFAULT_USER_EMAIL } from "../../utilities";
 import { DataSource, UpdateResult } from "typeorm";
 import { DataModelService, Student, User } from "@sims/sims-db";
 import { UserLoginInfo } from "./user.model";
+import { SERVICE_ACCOUNT_DEFAULT_USER_EMAIL } from "@sims/utilities";
```

Comment:
Isn't `SYSTEM_USER_DEFAULT_EMAIL` be more suitable?

Reply:
it was already there, i just moved to libs

Reply:
I do not have a strong reason to change it.